exonum/src/runtime/dispatcher/migration_tests.rs

```diff
@@ -364,9 +375,11 @@ fn test_migration_workflow(freeze_service: bool) {
 
     // Now, the migration start should succeed.
     let fork = rig.blockchain.fork();
-    rig.dispatcher()
+    let ty = rig
```

Comment:
`ty` 🙃 I've never seen such abbreviation before

Reply:
It's used, for example, in [`syn`](https://docs.rs/syn/1.0.14/syn/struct.Field.html#structfield.ty) because `type` is a reserved keyword.